crates/astria-conductor/src/executor/mod.rs

```diff
@@ -189,14 +196,21 @@ impl<C: ExecutionClient> Executor<C> {
         Ok(())
     }
 
-    /// checks for relevant transactions in the SequencerBlock and attempts
-    /// to execute them via the execution service function DoBlock.
-    /// if there are relevant transactions that successfully execute,
-    /// it returns the resulting execution block hash.
-    /// if the block has already been executed, it returns the previously-computed
-    /// execution block hash.
-    /// if there are no relevant transactions in the SequencerBlock, it returns None.
+    /// Send a block to the execution layer for execution.
+    ///
+    /// This function checks for relevant transactions in the SequencerBlock and attempts
+    /// to execute them via the execution service function DoBlock. If there are relevant
+    /// transactions that successfully execute,it returns the resulting execution block hash. If
+    /// the block has already been executed, the previously-computed execution block
+    /// hash is returned. If there are no relevant transactions in the SequencerBlock, it returns
+    /// None. If multiple blocks end up being executed (because several blocks were in
+    /// the queue waiting for a missing block to arrive), only the execution
+    /// hash of the last block is returned. The execution hash of all executed
+    /// blocks are stored in the `sequencer_hash_to_execution_hash` map.
     async fn execute_block(&mut self, block: SequencerBlockSubset) -> Result<Option<Vec<u8>>> {
+        // TODO: (GHI 285 - https://github.com/astriaorg/astria/issues/285): decide how empty blocks should be handled. either do nothing,
+        // or execute an empty block.
+        // if there are no transactions in the block, return None
```

Comment:
I don't see a need for executing empty blocks - I think we can ignore them, unless someone sees a good use case for empty rollup blocks?